front/migrations/20241211_parents_migrator.ts

```diff
@@ -190,7 +190,60 @@ const migrators: Record<ConnectorProvider, ProviderMigrator | null> = {
   },
   google_drive: null,
   microsoft: null,
-  github: null,
+  github: {
+    transformer: (nodeId, parents) => {
+      const repoId = parents[parents.length - 1];
```

Comment:
Can we check that this one looks like a repoId (numbers only)?

Reply:
yes i'll do that

Reply:
done